### ISSUE TITLE
No way to read Light Spectrum Dialog with Jaws

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#261]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/461. On Jaws.
I was unable to get Jaws to read the dialog in Show Light Spectrum. I had no trouble getting it to read the keyboard nav dialog.

**Visuals**
https://drive.google.com/file/d/1C14mC5cqm_eWSMtquM31HYOsBoShdfY9/view?usp=sharing

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.21/phet/molecules-and-light_en_phet.html?ea&a11y&supportsSound
Version: 1.5.0-dev.21 2019-12-13 19:43:51 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
In https://github.com/phetsims/molecules-and-light/issues/245 I noticed that the instrumentation of the dialog looks incorrect, ill fix up and see if that resolves the issue with JAWS

**Comment by jessegreenberg** [comment]
Thanks for reporting @KatieWoe I verified that this is fixed after #245
